Source/Redemption/Private/MovingPlatform.cpp

```diff
@@ -51,7 +57,7 @@ void AMovingPlatform::CheckMoveConditions()
 
 void AMovingPlatform::Move(float DeltaTime)
 {
-	if (IsMoving)
+	if (IsMoving || !NeedKey)
```

Comment:
They are connected to the same key system we have for the gates, am I understanding that correctly?

Reply:
yes